src/form-schema.js

```diff
@@ -154,6 +164,11 @@ export default class FormSchema extends FormFieldSchema {
       this.addSystemColumn('Edited Duration', 'editedDuration', '_edited_duration', 'integer');
     }
 
+    if (this.hasRecordKey()) {
+      this.addSystemColumn('Record Key', 'recordKey', '_record_key', 'string');
+      this.addSystemColumn('Record Sequence', 'recordSequence', '_record_sequence', 'integer');
```

Comment:
do we also need to check for the _record_sequence column as well? Or can we safely assume if _record_key is there, _record_sequence is also there?